device-types/MikroTik/RB1200.yaml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: hARB1200P
```

Comment:
The model name according to the website is `RB1200`. Please remove the leading `hA`, as well as the training `P` from the model

Reply:
sorry my mistake.